projects/court-case-and-delius/src/main/resources/application.yml

```diff
@@ -72,7 +74,9 @@ spring.datasource.url: jdbc:h2:mem:./test;MODE=Oracle;DEFAULT_NULL_ORDERING=HIGH
 
 ---
 spring.config.activate.on-profile: oracle
-spring.datasource.url: 'jdbc:tc:oracle:slim-faststart:///XEPDB1'
+spring.datasource:
+  url: 'jdbc:tc:oracle:slim-faststart:///XEPDB1'
+  jpa.hibernate.ddl-auto: create
```

Comment:
Do we need the `ddl-auto: create` ?

Reply:
Good spot, no idea where that came from - removed!